### DISCUSSION TITLE
need help with the ingress service

### DISCUSSION THREAD

**Discussion by mhazan01** [zifeo/terraform-openstack-rke2#40] [Q&A] [answered]
Hello,
i'm not quite sure i understand the ingress part well. is cilium or kube-vip supposed to provide the ingress service ? or do i need to deploy  one after the cluster is setup ?

**Comment by zifeo** [comment, marked as answer]
RKE2 by default comes with ingress-nginx, however in this module we decided to disable it and let the user deploy his own ingress controller manually (ingress-nginx, contour, etc.) so it can be customized to each use case.

**Reply by mhazan01** [reply]
make sense, thanks !
